lib/puppet/type/cron.rb

```diff
@@ -18,29 +18,29 @@
 
     Example:
 
-        cron { logrotate:
-          command => "/usr/sbin/logrotate",
-          user    => root,
-          hour    => 2,
-          minute  => 0
+        cron { 'logrotate':
+          command => '/usr/sbin/logrotate',
+          user    => 'root',
+          hour    => '2',
+          minute  => '0',
```

Comment:
What's the deal with quoting these numbers? They ain't octal, so it should be fine to use real integers, right?